resources/views/projects/tasks/create.blade.php

```diff
@@ -0,0 +1,83 @@
+@extends('layouts.app')
```

Comment:
Do we need a completely new view for this? It's 3 fields, seems like something you could modal into a task list view

Reply:
En esta view eventualmente también estará la lista de project collaborators para poder asignarles el task (#49) y entonces creo que se va a ir llenando mucho este view como para que sea un modal. 

Lo estoy resolving por ahora pero si crees que es mejor como un modal aún así, please unresolve this comment. :)

Reply:
I do, sobre todo porque considera el use case de agregar m&uacute;ltiples tasks nuevas. It's gonna be a huge pain to be constantly reloading pages y el problema de los collaborators se resuelve f&aacute;cilmente con un fuzzy dropdown